### PR TITLE
ID-433 adding postcode lookup to sirius mock#patch

### PR DESCRIPTION
**Pull request by spinhalf-com**
## Purpose

_Briefly describe the purpose of the change._

Fixes ###-####

## Approach

_Explain how your code addresses the purpose of the change._

## Learning

_Any tips and tricks, blog posts or tools which helped you._

## Checklist

* [ x] I have performed a self-review of my own code
* [ x] I have added relevant logging with appropriate levels to my code
* [ x] I have updated documentation where relevant
* [ x] I have added tests to prove my work
* [ x] I have run an accessibility tool against the changes and applied fixes
* [ ] The team have tested these changes

### CODE REVIEW

File: docker/sirius/openapi.yml [hunk 1]
```diff
@@ -73,24 +73,24 @@ paths:
...
                     postcode:
                       type: string
                       x-faker:
-                        helpers.replaceSymbols: "??# #??"
+#                        helpers.replaceSymbols: "??# #??"
                     description:
                       type: string
                       x-faker:
-                        fake: "A concatenation of the address lines"
+#                        fake: "A concatenation of the address lines"
```
Comment (mohammedrmoj) [inline docker/sirius/openapi.yml:93]: Maybe just delete these?

File: docker/sirius/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,4 @@
 FROM outofcoffee/imposter:4.2.1
 
 COPY . /opt/imposter/config
+
```
Comment (mohammedrmoj) [inline docker/sirius/Dockerfile:4]:
```suggestion
```

**Review by mohammedrmoj — DISMISSED** [review]
Looks good. Recommend removing commented out stuff

File: docker-compose.yml [hunk 3]
```diff
@@ -12,6 +12,8 @@ services:
       retries: 3
     volumes:
       - ./docker/sirius:/opt/imposter/config
+    ports:
+      - 8081:8080
```
[the remark this answers is not in the document]
Comment (spinhalf-com) [reply docker-compose.yml:16]: Sorry, used only for testing, will remove

**Review by gregtyler — APPROVED** [review]

**Pull request merged**